src/services/customerServies.js

```diff
@@ -24,9 +27,35 @@ const deleteCustomer = async id => {
     return deleteCustomer
 }
 
+const customerReport = async options => {
```

Comment:
I'm thinking of having a one utility function that does this logic of getting the records number reports